pyicloud/base.py

```diff
@@ -80,7 +82,7 @@ def request(self, method, url, **kwargs):  # pylint: disable=arguments-differ
                 api_error = PyiCloudAPIResponseException(
                     response.reason, response.status_code, retry=True
                 )
-                request_logger.warn(api_error)
+                request_logger.warning(api_error)
```

Comment:
@Quentame - fyi `There is an obsolete function warn which is functionally identical to warning. As warn is deprecated, please do not use it - use warning instead.`

see https://docs.python.org/3/library/logging.html

Reply:
Yep, seen but not fixed 🙄 

Thanks !